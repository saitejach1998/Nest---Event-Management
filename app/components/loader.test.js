import React from 'react';
import Loader  from './loader.js';

import renderer from 'react-test-renderer';

describe('Loader  component', function() {
    it('renders a SignupCompont ', function() {
      const tree = renderer.create(
        <Loader  />
        ).toJSON();
    expect(tree).toMatchSnapshot();
    });
  });