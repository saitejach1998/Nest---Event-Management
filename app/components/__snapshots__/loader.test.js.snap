// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader  component renders a SignupCompont  1`] = `
<Modal
  animationType="none"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00000040",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "space-around",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 10,
          "display": "flex",
          "height": 100,
          "justifyContent": "space-around",
          "width": 100,
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#999999"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
  </View>
</Modal>
`;
